datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -113,6 +115,14 @@ public FlowIdentifier triggerSdxResize(Long sdxClusterId, SdxCluster newSdxClust
                 environmentClientService.getBackupLocation(newSdxCluster.getEnvCrn()), performBackup));
     }
 
+    public FlowIdentifier triggerSdxResizeRecovery(SdxCluster oldSdxCluster, SdxCluster newSdxCluster) {
+        LOGGER.info("Triggering recovery for failed SDX resize with original cluster: {} and resized cluster: {}",
+                oldSdxCluster, newSdxCluster);
+        String userId = ThreadBasedUserCrnProvider.getUserCrn();
```

Comment:
nit: can we call this `userCrn` if it's not id but crn?

Reply:
So this use of `userId` is in basically every single other method within this file. Would you like me to change all of those as well?